pkg/manager/manager.go

```diff
@@ -169,7 +169,8 @@ type Manager struct {
 	SupervisorSet    *supervisor.SupervisorsSet
 
 	// A basic configuration template loaded from the file
-	DaemonConfig daemonconfig.DaemonConfig
+	DaemonConfig     daemonconfig.DaemonConfig
+	MirrorsConfigDir string
```

Comment:
`manager` package is responsible to manage all nydusd daemons' lifecycles. I am afraid it's not suitable to associate the mirrors config to daemons' manager

Reply:
Replaced it with global config.